tasks/agent.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Add Icinga APT key
+  apt_key:
+    url: https://packages.icinga.com/icinga.key
```

Comment:
I think you should install `apt-transport-https` for this

Reply:
No, Ansible handles that

Reply:
Also, you need `apt-transport-https` for https apt repos, this is just a key. Basically Ansible executes something like `curl -L https:/... | apt-key add -`